proto/cronos/cronos.proto

```diff
@@ -0,0 +1,23 @@
+syntax = "proto3";
+package cryptoorgchain.cronos.cronos;
+
+import "gogoproto/gogo.proto";
+
+option go_package = "github.com/crypto-org-chain/cronos/x/cronos/types";
+
+// Params defines the parameters for the cronos module.
+message Params {
+    option (gogoproto.goproto_stringer)             = false;
+    repeated ConvertEnabled convert_enabled         = 1 [(gogoproto.moretags) = "yaml:\"convert_enabled,omitempty\""];
+    string                  ibc_cro_denom           = 2 [(gogoproto.moretags) = "yaml:\"ibc_cro_denom,omitempty\""];
+    string                  ibc_cro_channelID       = 3 [(gogoproto.moretags) = "yaml:\"ibc_cro_channelID,omitempty\""];
```

Comment:
Is this duplicated, I think we can query channel id from the ibc token denom?

Reply:
good point, I will remove it